x-pack/solutions/observability/plugins/streams/server/routes/streams/ingest/route.ts

```diff
@@ -41,31 +32,23 @@ const readIngestRoute = createServerRoute({
     path: z.object({ id: z.string() }),
   }),
   handler: async ({ params, request, getScopedClients }): Promise<IngestGetResponse> => {
-    try {
-      const { streamsClient } = await getScopedClients({
-        request,
-      });
-
-      const name = params.path.id;
+    const { streamsClient } = await getScopedClients({
+      request,
+    });
 
-      const definition = await streamsClient.getStream(name);
+    const name = params.path.id;
 
-      if (isWiredStreamDefinition(definition)) {
-        return { ingest: definition.ingest };
-      }
+    const definition = await streamsClient.getStream(name);
 
-      if (isUnwiredStreamDefinition(definition)) {
-        return { ingest: definition.ingest };
-      }
-
-      throw badRequest(`Stream is not an ingest stream`);
-    } catch (e) {
-      if (e instanceof DefinitionNotFound || (isResponseError(e) && e.statusCode === 404)) {
-        throw notFound(e);
-      }
+    if (isWiredStreamDefinition(definition)) {
+      return { ingest: definition.ingest };
+    }
 
-      throw internal(e);
+    if (isUnwiredStreamDefinition(definition)) {
+      return { ingest: definition.ingest };
     }
+
+    throw badRequest(`Stream is not an ingest stream`);
```

Comment:
So it's still fine for a route to directly throw a Boom, rather than a "domain" (StatusError) error that gets translated?

Reply:
yes, or do you see a reason not to allow that?

Reply:
No, just looking to understand plugin philosophy :)